crates/tap-agent/src/agent/sender_allocation.rs

```diff
@@ -970,23 +973,26 @@ pub mod tests {
             sender_aggregator,
             config: super::AllocationConfig {
                 timestamp_buffer_ns: 1,
-                rav_request_receipt_limit: 1000,
+                rav_request_receipt_limit,
                 indexer_address: INDEXER.1,
                 escrow_polling_interval: Duration::from_millis(1000),
             },
         }
     }
 
+    #[bon::builder]
     async fn create_sender_allocation(
         pgpool: PgPool,
         sender_aggregator_endpoint: String,
         escrow_subgraph_endpoint: &str,
+        rav_request_receipt_limit: Option<u64>,
         sender_account: Option<ActorRef<SenderAccountMessage>>,
     ) -> (ActorRef<SenderAllocationMessage>, Arc<Notify>) {
```

Comment:
You can remove all the calls to `sender_aggregator_endpoint` that contains DUMMY_URL.
```suggestion
    async fn create_sender_allocation(
        pgpool: PgPool,
        #[builder(default = DUMMY_URL.to_string())]
        sender_aggregator_endpoint: String,
        escrow_subgraph_endpoint: &str,
        rav_request_receipt_limit: Option<u64>,
        sender_account: Option<ActorRef<SenderAccountMessage>>,
    ) -> (ActorRef<SenderAllocationMessage>, Arc<Notify>) {
```

Reply:
any updates here?